src/sonic-config-engine/sonic-cfggen

```diff
@@ -331,6 +331,14 @@ def main():
         else:
             deep_update(data, parse_xml(minigraph, port_config_file=args.port_config, asic_name=asic_name, hwsku_config_file=args.hwsku_config))
 
+        # enable TACACS per-command accounting by default on 202012 branch
```

Comment:
sonic-cfggen may be not proper place to add the default value. For example, in [L2-Switch-mode](https://github.com/sonic-net/SONiC/wiki/L2-Switch-mode), there is no minigraph.xml at all, and there is no need to set the accounting default value.

Reply:
Offline discussion, this is OK